casbin/model/policy.py

```diff
@@ -85,10 +85,29 @@ def add_policies(self,sec,ptype,rules):
     def update_policy(self, sec, ptype, old_rule, new_rule):
         """update a policy rule from the model."""
 
-        if not self.has_policy(sec, ptype, old_rule):
+        if sec not in self.model.keys():
+            return False
+        if ptype not in self.model[sec]:
+            return False
+
+        ast = self.model[sec][ptype]
+
+        # find rule index, if fail return false
+        try:
+            rule_index = ast.policy.index(old_rule)
+        except:
             return False
```

Comment:
plz use `in` operator

Reply:
I think it will be more efficient in this way ,it noly find once if the rule exit in the policy. But it's don't influence the time complexity, and I will change it for pythonic, thanks.